### DISCUSSION TITLE
[enhancement] continous learning for Tabular Predictor

### DISCUSSION THREAD

**Discussion by mglowacki100** [autogluon/autogluon#3420] [Ideas]
Hi,
Autogluon support contionous learning (aka incremental learning) for `Multimodal Predictor` (see https://auto.gluon.ai/stable/tutorials/multimodal/advanced_topics/continuous_training.html) but if you want to do the same thing for `Tabular Predictor` you'll get error that model is already fitted.
I think the same functionality should be possible for "tabular" as xgboost, catboost and lightgbm supports incremental learning (see example here: https://towardsdatascience.com/you-dont-need-neural-networks-to-do-continual-learning-2ed3bfe3dbfc), but stacking could be tricky. Maybe it could be implemented by experimental preset 'continual_learning' that will confine list of models to models that support continous learning?

**Comment by Killerkk** [comment]
+1

**Comment by celestinoxp** [comment]
@Innixma
